src/test/resources/com/puppycrawl/tools/checkstyle/checks/javadoc/javadocmethod/InputJavadocMethodParamsTags.java

```diff
@@ -0,0 +1,18 @@
+package com.puppycrawl.tools.checkstyle.checks.javadoc.javadocmethod;
+
+public class InputJavadocMethodParamsTags {
+
+  /**
+   * A method with an undocumented param and a missing param.
+   * Checkstyle should at least
+   * missingParam, ideally would be nice to have a flag to report
+   * also undocumentedParam
+   *
+   * @param param1 some text
+   * @param unexpectedParam
+   * @return
```

Comment:
Please add an example of a 2nd parameter where the description is on the next line and not the same line as `@param`.

Reply:
added more tests